.github/workflows/update-changelog.yaml

```diff
@@ -0,0 +1,38 @@
+name: "Update changelog"
```

Comment:
Why do we need this?

Reply:
I think this came from backport PR at the beginning.

Reply:
This file should be removed

Reply:
I think the best way to fix this small of a backport PRs is to simply redo the changes, not use `git`